src/Loggers/LogImage.jl

```diff
@@ -1,16 +1,35 @@
+
+@enum ImageFormat HW WH HWC WHC CHW CWH HWN WHN NHW NWH HWCN WHCN CHWN CWHN NHWC NWHC NCHW NCWH
```

Comment:
I think it would be good to export ImageFormat.

Reply:
i have exported `ImageFormat` in `TensorBoardLogger.jl`